install.py

```diff
@@ -138,6 +155,23 @@ def launch_dockers(self):
         print('Diffgram Successfully Launched!')
         print('View the Web UI at: http://localhost:8085')
 
+    def set_diffgram_version(self):
+        version = input('Enter diffgram version: [Or Press Enter to Get The Latest Version]: ')
+        if version == "":
+            self.diffgram_version = 'latest'
+        else:
+            self.diffgram_version = 'version'
```

Comment:
We want variable not string here maybe?

Reply:
You're right thanks for catching that one!